### PR TITLE
Updating analytics to use build server

### PR DESCRIPTION
**Pull request by blckmn**
- allows for server side analytics submission to whatever service we plan to use

### CODE REVIEW

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> FAIL
 - `Tested` label found -> FAIL
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

**Review by nerdCopter — APPROVED** [review]
* approving as nothing seemingly broken; tested flashing FC and configuring.
* did not inspect the analytics code

File: src/js/Analytics.js
Comment (haslinghuis) [inline src/js/Analytics.js:32]: Using `const` was okay here

**Review by haslinghuis — APPROVED** [review]

**Review by haslinghuis — APPROVED** [review]

**Pull request merged**
